chunky/src/java/se/llbit/chunky/renderer/scene/Scene.java

```diff
@@ -1397,11 +1411,7 @@ private int calculateOctreeOrigin(Collection<ChunkPosition> chunksToLoad) {
     int maxDimension = Math.max(yMax - yMin, Math.max(xmax - xmin, zmax - zmin));
     int requiredDepth = QuickMath.log2(QuickMath.nextPow2(maxDimension));
 
-    int xroom = (1 << requiredDepth) - (xmax - xmin);
-    int yroom = (1 << requiredDepth) - (yMax - yMin);
-    int zroom = (1 << requiredDepth) - (zmax - zmin);
-
-    origin.set(xmin - xroom / 2, -yroom / 2, zmin - zroom / 2);
```

Comment:
This probably shouldn't have been removed.

Reply:
this piece of code made it so the loaded chunks where place at the middle of the octree. 
For example if you load a 3\*3 chunks area (with a small ymax-ymin), you have a 48\*48 blocks area so you need a octree with depth 6 ie, a 64\*64 blocks octree. With this piece of code, the 48\*48 area area is placed in the [8; 56]\*[8; 56]. Without this piece of code, it is placed in the [0; 48]\*[0; 48] area.
Visually this makes no difference, the difference is that without this translation, chunks are aligned with the nodes of the octree so it is possible to load a whole 16\*16\*16 blocks at a time, if they weren't aligned this would not be possible. (In this case, the piece of code made it so that it would at least be 8-aligned instead of 16-aligned so it would still be possible to set 8\*8\*8 blocks at a time but the more we can do at a time, the better)

Reply:
Would this then break existing scenes when reloading the chunks?

Reply:
Sorry for the delay. It breaks existing when loading them (the octree is shifted somewhere else) but reloading the chunks refixes it.
If I want my change and be backward compatible maybe I should store the origin so it is not recomputed when loading an existing scene (and it is recomputed using the old version is the origin information is not there)

Reply:
No problem. I'm fine with having the octree origin not at its center. Maybe we can just bump the octree version and move the old octree when it is loaded?
This won't affect entity positions, right?

Reply:
For now, only the octree is shifted when loading an existing scene, entities are still correctly relative to the camera (so they are not where they should be relative to the octree)
Looking at the code, I see that the origin is used to place emitter position as well, I don't how if they are shifted as well or not for now